mhs/outbound/outbound/transmission/outbound_transmission.py

```diff
@@ -49,9 +49,14 @@ def __init__(self, certs_dir: str, client_cert: str, client_key: str, ca_certs:
             try:
                 logger.info("0001", "About to send message with {headers} to {url} : {message}",
                             {"headers": headers, "url": url, "message": message})
+                # ******************************************************************************************************
+                # TLS CERTIFICATE VALIDATION HAS BEEN TEMPORARILY DISABLED! This is required because Opentest's SDS
+                # instance currently returns endpoints as IP addresses. This MUST be changed before this code is used in
+                # a production environment
+                # ******************************************************************************************************
                 response = await CommonHttps.make_request(url=url, method=request_method, headers=headers, body=message,
                                                           client_cert=self._client_cert, client_key=self._client_key,
-                                                          ca_certs=self._ca_certs)
+                                                          ca_certs=self._ca_certs, validate_cert=False)
```

Comment:
I wasn't looking to make this too complicated, since it's really a workaround until Opentest is fixed, and I don't think it's necessarily something we need to support after that.

I'll add a note to the README to make it explicitly clear that the current version of the MHS isn't doing certificate validation.